airflow-core/src/airflow/example_dags/example_assets.py

```diff
@@ -50,6 +50,7 @@
 automatically as they depend on assets that do not get updated or are not produced by any scheduled tasks.
 """
 
+# [START asset_def]
```

Comment:
Its ok to move the marker to after "from future" imports

Reply:
Changed it.